### ISSUE TITLE
Final Report Review

### ISSUE THREAD

**Issue by sesmalling** [GraceJiangJinyi/ORIE4741_Project#14]
The team set out to predict how stocks performed in the US and China using COVID-19 case information. The team used linear regression, as well as quantile regression, support vector machines, and neural networks to create models that might help to perform these predictions. After trying many models, a few performed well with very low MAE and MSE values, while some were much more prone to overfitting. 

I really appreciated the overall organization of the report, with models increasing in complexity as the report went on. Additionally, although a little less formal, the bullet points and short paragraphs made the report more readable. Lastly, the short conclusion points at the end of each model were informative to a reader interested in improving portfolio performance or just interested in overall trends included in the data. 

Although readability was improved with the bullet points, I found many of the graphs and tables difficult to read, with confusing labels and too many values. I think that highlighting a few values, especially in the tables with MSE and MAE values, would be very helpful to show the reader which values were the most important to see. Along the same lines, it would have been more helpful to have numbered the figures and tables so they are easier to refer to. Overall, I thought your team's report was well thought-through, with concise conclusions and a good progression of modeling attempts.
